static/bidder-params/revcontent.json

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Nitpick: Please remove the leading line break.

Reply:
whoops.